components/footer.js

```diff
@@ -0,0 +1,79 @@
+import styled from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import dashboardIcon from "../public/dashboard_icon.png";
+import expensesIcon from "../public/expenses_icon.png";
+import revenuesIcon from "../public/revenues_icon.png";
+import profileIcon from "../public/profile_icon.png";
+
+export default function Footer() {
+  const { pathname } = useRouter();
+
+  return (
+    <>
```

Comment:
I think you don't need the `Fraction` here. `StyledFooter` is the parent element.